src/db/users.js

```diff
@@ -0,0 +1,32 @@
+var records = [
+  {
+    id: 1,
+    username: "user",
+    password: "password",
+    displayName: "John",
+    physicalLevel: 3,
+  },
+];
+
+exports.findById = function (id, cb) {
+  process.nextTick(function () {
+    var idx = id - 1;
+    if (records[idx]) {
```

Comment:
```suggesion
     if (records.hasOwnProperty(idx)) { 
```

Reply:
Looking closer this whole thing gets reworked when stating UserID at 0. Updating if statement as suggested and removing the var idx and just using id instead.

Reply:
Oh I think this is easier to read than the implicit boolean check. If there was a `null` in the array this would return true now where your way would return false but besides that they are the same.

Reply:
I'm curious whats up with the `process.nextTick` though? I normally use promises, but afaict making this logic run immediately would be fine.